ci-operator/config/Apicurio/apicurio-registry-system-tests/Apicurio-apicurio-registry-system-tests-main__service-registry-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,41 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+  os:
+    name: ubi
+    namespace: ocp
+    tag: "8"
+  test-image:
+    name: apicurio-ci-tools
+    namespace: ci
+    tag: interop
+releases:
+  latest:
+    candidate:
+      product: ocp
+      stream: nightly
+      version: "4.13"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: service-registry-2-3
+  cron: 0 6 * * 1
+  remote_api: true
+  steps:
+    cluster_profile: aws-cspi-qe
+    env:
+      BASE_DOMAIN: cspilp.interop.ccitredhat.com
+    test:
+    - ref: service-registry-run-tests
+    workflow: ipi-aws
+  timeout: 4h0m0s
+zz_generated_metadata:
+  branch: main
+  org: Apicurio
+  repo: apicurio-registry-system-tests
+  variant: service-registry-ocp4.13-lp-interop
```

Comment:
Change 4.13 to 4.14 in the filename of this config. Then once you run `make update` after that this metadata will be updated.